InTune/InTune/ViewControllers/ProfileViewController.swift

```diff
@@ -9,22 +9,19 @@
 import UIKit
 
 class ProfileViewController: UIViewController {
-
+    
+    @IBOutlet weak var profImage: UIImageView!
```

Comment:
Outlets can be set to `strong` instead of `weak` for example `@IBOutlet private var profImage: UIImageView!`. Outlet should be marked as `private` as the ViewController class should only be responsible for any modification.  

A refresher article for `strong` instead of `weak` outlet https://cocoacasts.com/should-outlets-be-weak-or-strong

Reply:
Got it, I'll update that, thank you!